sql/core/src/test/resources/sql-tests/inputs/literals.sql

```diff
@@ -107,7 +107,11 @@ select integer '2147483648';
 
 -- awareness of the negative/positive sign before type
 select -integer '7';
+select +integer '7';
+select +date '1999-01-01';
```

Comment:
We should have a followup to forbid this. I've checked pgsql and it fails.

Reply:
do we have a typical identifier that can match `+date` as a whole?

Reply:
no, but the parser simply ignores the `+` sign.